packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -355,6 +356,7 @@ export default {
         }
 
         .button-wrapper {
+            display: inline-block;
```

Comment:
I think properties are inherited if not specified, so we may not need to include this in the media query. Was it included for easy of maintenance or?

Reply:
It was included in the previous implementation and I just reverted it. Seems like this is not needed 😅 Anyway I have shifted this to the minimal panel component.